spec/controllers/account_reset/delete_account_controller_spec.rb

```diff
@@ -29,6 +29,26 @@
       expect(response).to redirect_to account_reset_confirm_delete_account_url
     end
 
+    it 'logs a good token to the attempts api' do
+      user = create(:user, :signed_up, :with_backup_code)
+      create(:phone_configuration, user: user, phone: '+1 703-555-1214')
+      create_list(:webauthn_configuration, 2, user: user)
+      create_account_reset_request_for(user)
+      grant_request(user)
+
+      session[:granted_token] = AccountResetRequest.all[0].granted_token
```

Comment:
I prefer `.first` to `[0]` personally for readability
```suggestion
      session[:granted_token] = AccountResetRequest.first.granted_token
```